src/app/_utils/s3Utils.ts

```diff
@@ -0,0 +1,194 @@
+import { CardData } from "../_components/_sharedcomponents/Cards/CardTypes";
+
+// Deck data interfaces for deck info from swudb
+interface DeckMetadata {
```

Comment:
nit: the convention we use in the forceteki repo is that all interface and type names start with "I" (e.g. IDeckMetadata), not sure if y'all want to use the same convention here

Reply:
that might be a good convention to have

Reply:
changed the names of most of the interfaces. Except for those that do overrides.